HTML to PDF/Blink/Converting_SVG_to_Image_using_HtmlConverter/.NET/Converting_SVG_to_Image_using_HtmlConverter/Program.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Syncfusion.Drawing;
+using Syncfusion.HtmlConverter;
+
+HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+string svg = File.ReadAllText(Path.GetFullPath(@"Data/Input.svg"));
+Image images = htmlConverter.ConvertToImage(svg, "");
+byte[] imageByte = images.ImageData;
```

Comment:
Image image = htmlConverter.ConvertToImage(svg, "");
byte[] imageBytes = image.ImageData;
//Save the image
File.WriteAllBytes(Path.GetFullPath(@"Output/Output.jpg"), imageBytes);

Reply:
Modified